pkg/cmd/pipeline/pipeline.go

```diff
@@ -31,7 +31,10 @@ func Command(p cli.Params) *cobra.Command {
 	}
 
 	flags.AddTektonOptions(cmd)
-	cmd.AddCommand(listCommand(p))
-	cmd.AddCommand(describeCommand(p))
+	cmd.AddCommand(
```

Comment:
👍 
would be nicer if we sorted it

Reply:
You meant to say that inside AddCommand I should pass parameter in alphabetical order?